proxy/src/serverless/http_util.rs

```diff
@@ -45,6 +45,10 @@ pub fn api_error_into_response(this: ApiError) -> Response<Full<Bytes>> {
             err.to_string(),
             StatusCode::REQUEST_TIMEOUT,
         ),
+        ApiError::Cancelled => HttpErrorBody::response_from_msg_and_status(
+            "Request cancelled".to_string(),
```

Comment:
```suggestion
            this.to_string(),
```
The same as my former comment

Reply:
s/this/self/ like in the other, but thanks for chiming in.